### PR TITLE
Add V8 v8.1 blog post

### PR DESCRIPTION
**Pull request by dinfuehr**
At least the Tweet ID is still missing.

### CODE REVIEW

File: src/blog/v8-release-81.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: 'V8 release v8.1'
+author: 'Dominik Inführ'
```
Comment (RReverser) [inline src/blog/v8-release-81.md]: Please add a tagline.

File: src/blog/v8-release-81.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+avatars:
+  - 'dinfuehr'
```
Comment (RReverser) [inline src/blog/v8-release-81.md]: I don't see the referenced avatar files in the PR. Seems like they were omitted from `git add`?

Comment (dinfuehr) [reply src/blog/v8-release-81.md]: Is there a script I need to run for this?

Comment (RReverser) [reply src/blog/v8-release-81.md]: No, you need to manually add images with given name to [src/img/_avatars](https://github.com/v8/v8.dev/tree/master/src/_img/avatars) folder.

These should be 2 variants of your avatar with same format, dimensions and comparable file sizes as other avatars in that folder (check them as a guide). You can use [Squoosh.app](https://squoosh.app/) for image compression.

File: src/blog/v8-release-81.md (hunk 1 above)
Comment (RReverser) [reply src/blog/v8-release-81.md]: @dinfuehr Ping - I believe this is the only issue left :)

**Comment by dinfuehr** [comment]
Added tagline, thanks to @LeszekSwirski for that ;)

**Pull request merged**